### DISCUSSION TITLE
Make >ON AIR< Sign changeable?

### DISCUSSION THREAD

**Discussion by JohnR98** [cpvalente/ontime#1297] [Ideas]
Hi, was it possible to create an input field, to change the Words >ON AIR< in Action, Showtime or anything else? Br

**Comment by cpvalente** [comment]
Hi @JohnR98 , we have been tied up with the preparations to the release of the cloud version of Ontime and have trailed back on replying to users

This is not currently possible, you could change this using CSS if you need or we could create a issue ticket for a feature request if you feel this is important

**Comment by pbelbin** [comment]
It would be really good to be able to, for some events, to change the layout so that instead of the clock being the large focus point, with the text of the current/next beneath, to be able to show large text where the clock is shown.
eg: for the first so many seconds of a show, it would be useful to show the words 'You're Live', for example.

**Reply by alex-Arc** [reply]
Hi @pbelbin 
Have you seen the design for the from the v4 alpha
<img width="1910" height="1036" alt="image" src="https://github.com/user-attachments/assets/1d168576-282c-4d83-8d54-a8bd88a6da0a" />

dose that change what you are thinking of

**Comment by pbelbin** [comment]
Thank you for the reply.

 

Saw the image you shared.

 

Is the intention that this be something talent would see, or is it more for the person configuring the rundown?

 

What I’m looking for is something that shows the time remaining as it runs down to 0, and then, the next event is simply a text-only event, so that there’s no confusion by the talent as to what/where they are supposed to be seeing an indication that they’re now ‘live’.

 

Eg: have an event that’s the count down to 0:00, then, the next event is just showing text only, of ‘You’re live’ or ‘ON AIR’ (or other custom text) for the duration of that event.

 

The ‘text only’ event would be set up to be shown for say, 10 seconds, and the following event might revert back to showing a clock, or, might be another text-only event, showing some additional information, and perhaps then the next event might be the clock with current and next event names.

 

Regards,

 

Peter

**Reply by alex-Arc** [reply]
Is this not available in the normal timer view ?
or the title is to small?

**Comment by pbelbin** [comment]
Not sure what you're referring to.

Admittedly, I was able to configure a web browser timer to display, but,
this is in a web browser, and produces a URL with various options, like,
the text to display when the current timer expires.

From what I can see, this is *not* part of the rundown, so it's kind of a
one-off thing, not something that's just part of the rundown.

If what you're referring to is something that's part of the rundown
definition, sounds good, but, that's not what I was able to get to work.

Perhaps a bit more of an example would be useful?

Regards,

Peter

**Reply by alex-Arc** [reply]
If you take a look at the timer page, it follows where you are in the rundown
https://docs.getontime.no/interface/backstage/stage-timer/

or could you illustrate what you mean by "part of the rundown"

**Comment by pbelbin** [comment]
The issue is that the numbers are very big, and the title of the rundown item is relatively small, and someone that’s not familiar, or ‘in the moment’ may not be particularly focused on noticing what the title is.

 

It would be really great if, for example, when the *next* rundown item becomes active, be able to show text, large, scaled to occupy the time area, for a configurable amount of time, before it reverts to showing the time.

 

Eg: so, when it becomes active, large words will be seen for a configurable amount of time.

 

This would be a great way of highlighting that they’re now live, without them having to hunt around or be familiar.
